python/2FA/config.ini.default

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# Bandwith creds 
+# Account ID can be found in the upper right of your screen when you log in, in parens
+# the username and password for a user with API permissions
+[bandwidth]
+account_id = 9900910
+api_user = acovati-rnd
```

Comment:
You might want to take your creds out of here (assuming this will be publicly available)

Reply:
good catch, thank you.